dpdata/formats/lmdb/format.py

```
@@ -1,286 +1,2426 @@
+    @staticmethod
+    def _remap_legacy_type_table(
+        data: dict[str, Any],
+        specs: dict[str, _FieldSpec],
+        type_map: list[str],
+    ) -> None:
+        """Apply a requested type table to one reconstructed legacy system."""
+        old_names = _validate_names(data["atom_names"], "legacy atom_names")
+        requested = _validate_names(type_map, "type_map")
+        missing = [name for name in old_names if name not in requested]
+        active = {
+            name for name, count in zip(old_names, data["atom_numbs"]) if count > 0
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)